troposphere/static/js/collections/SizeCollection.js

```diff
@@ -9,7 +9,7 @@ define(function (require) {
   return Backbone.Collection.extend({
     model: Size,
 
-    url: globals.API_V2_ROOT + "/sizes",
+    url: globals.API_V2_ROOT + "/sizes?archived=true",
```

Comment:
You won't want this here, rather, you'll want to `fetchWhere` in the locations you would prefer archived sizes.

This line of code would allow 'deleted' sizes to show up in an instance launch, or anywhere else a 'size list' might show..